docs/content/docs/connectors/table/filesystem.md

```diff
@@ -328,15 +328,30 @@ Time extractors define extracting time from partition values.
         <td>String</td>
         <td>The extractor class for implement PartitionTimeExtractor interface.</td>
     </tr>
+    <tr>
+        <td><h5>partition.time-extractor.formatter-pattern</h5></td>
+        <td style="word-wrap: break-word;">yyyy-MM-dd&nbsp;HH:mm:ss</td>
+        <td>String</td>
+        <td>When 'partition.time-extractor.kind' is set to 'default', you can specify a formatter pattern to get a timestamp from partitions. <br>By default, the format is 'yyyy-MM-dd&nbsp;HH:mm:ss'.
```

Comment:
how about the name `timestamp-formatter` which has default value 'yyyy-MM-dd&nbsp;HH:mm:ss', and it can combine with `partition.time-extractor.timestamp-patter`, and it also should supports multiple partition fields like ```$year$month$day```.

Reply:
> And it can combine with partition.time-extractor.timestamp-patter, and it also should supports multiple partition fields like $year$month$day.

Is the above part of the description?